### PR TITLE
column names in [Data] now supports alternatives.

### PR DESCRIPTION
**Pull request by charles-cowart**

### CODE REVIEW

**Comment by charles-cowart** [comment]
Would you mind reviewing this PR @wasade since Antonio is out? Thanks! This should resolve Gail's issue this morning.

In brief, if a sample-sheet contains known variations on a column name in the [Data] section, it will issue a warning to the user and silently convert them to the standard defined in _KL_SAMPLE_SHEET_DATA_COLUMNS.

File: metapool/sample_sheet.py [hunk 1]
```diff
@@ -254,6 +255,47 @@ def _is_empty(csv_line):
...
+        # alternatives found in first sample will be true for all other
+        # samples as well.
+        if not self._samples:
+            # If this object has no samples, skip renaming of columns in [Data]
+            # section.
+            return
+
+        warning_msgs = []
+
+        obs = self._samples[0]._store
```
Comment (wasade) [inline metapool/sample_sheet.py:275]: This is interacting with a private member variable of another object defined in an external library. This is not safe to do. Why not modify `section_header` above?

**Review by wasade — CHANGES_REQUESTED** [review]

File: metapool/sample_sheet.py (hunk 1 above)
Comment (charles-cowart) [reply metapool/sample_sheet.py]: doh, it pays to look a little higher in the code, it seems. Thanks!

It is true, it's not safe to do, but at the time it looked like what needed to be done w/out modifying sample_sheets library directly.

**Comment by charles-cowart** [comment]
@wasade ready for review!

**Review by wasade — APPROVED** [review]
I think this is reasonable

**Comment by charles-cowart** [comment]
Thanks!

**Pull request merged**